### PR TITLE
Fix issue #845 and put some effort into postgres migrations

### PR DESCRIPTION
**Pull request by martin-bts**
* There are tons of old and unused plsql scripts in `askbot/search/postgres`. Moved to `old/` for clarity
* The migrations apply two plsql scripts and virtually immediately afterwards slightly different versions of those scripts, with no clear purpose. Using only most recent versions right away.
* Fixed definition of setup_aggregates() to avoid problems with Postgresql 11 as described in #845

### CODE REVIEW

File: askbot/search/postgresql/thread_and_post_models_03012016.plsql [hunk 1]
```diff
@@ -48,12 +48,8 @@ $$ LANGUAGE plpgsql;
 
 CREATE OR REPLACE FUNCTION setup_aggregates() RETURNS boolean AS
 $$
-DECLARE
-    onerow record;
 BEGIN
-    FOR onerow IN SELECT * FROM pg_proc WHERE proname = 'concat_tsvectors' AND proisagg LOOP
-        DROP AGGREGATE concat_tsvectors(tsvector);
-    END LOOP;
+    DROP AGGREGATE IF EXISTS concat_tsvectors(tsvector);
```
Comment (evgenyfadeev) [inline askbot/search/postgresql/thread_and_post_models_03012016.plsql]: yes this is better!

**Comment by evgenyfadeev** [comment]
Hey the fix on 845 here is good, I will it as you suggest, however I think won't be merging this PR, explaining the reason below.

While it's ok to make migrations running postgres scripts that no longer apply (only if you're 100% sure) 
 into a migration that just returns and does nothing, it is probably not a good idea to delete the files retroactively as there probably are sites that ran those migrations. To accept this I'd need a proof that this change won't break the migrations on the existing files.

+1 on deleting unused files.

I'll see if I can quickly reproduce this without changing the migration dependencies and in that case I will close this PR without merging it.

**Comment by martin-bts** [comment]
I do not know anything about Django migrations and I am unable to handle them. I am glad somebody else provided up to date migrations in https://github.com/ASKBOT/askbot-devel/pull/853 because I have been trying to reproduce these and Django would always tell me that it doesn't see any model changes. Therefore I cannot provide any proof for or against anything. We can keep the empty migrations if that makes any sense and only remove the unused files and adapt the current migrations to be Postgres 11+ compatible.

**Comment by evgenyfadeev** [comment]
I have incorporated the "DROP AGGREGATE IF EXISTS concat_tsvectors(tsvector);" in 6fd75c54470ced7f91d1077bd25d293a9b81974a  it's nice that you've stumbled on this.

Closing this as the issue of repeated scripts is now fixed.

**Pull request closed without merging**